tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -72,15 +72,6 @@ sub run_test {
 
         # SLES16 guest uses networkmanager to control network, no /etc/sysconfig/network/ifcfg*
         next if ($guest =~ /sles-16/i);
-        #Prepare the new guest network interface files for libvirt virtual network
-        #for some guests, interfaces are named eth0, eth1, eth2, ...
-        #for TW kvm guest, they are enp1s0, enp2s0, enp3s0, ...
-        my $primary_nic = script_output("ssh root\@$guest \"ip a|awk -F': ' '/state UP/ {print \\\$2}'|head -n1\"");
-        $primary_nic =~ /([a-zA-Z]*)(\d)(\w*)/;
-        for (my $i = 1; $i <= 6; $i++) {
-            my $nic = $1 . (int($2) + $i) . $3;
-            assert_script_run("ssh root\@$guest 'cp /etc/sysconfig/network/ifcfg-$primary_nic /etc/sysconfig/network/ifcfg-$nic'");
```

Comment:
Would you explain why this part not needed any more, even for sle15- vms?

Reply:
> Need not copy ifcfg-* files from the host to guests as they are created during network tests, besides the interface name is unknown at the moment yet

They are created during network tests now:
```
- if ((get_var('TEST', '') =~ m/qam-(kvm|xen)-install-and-features-test/ || $is_sriov_test eq "true") and !is_sle('16+')) {
+ unless (is_sle('16+')) {
        assert_script_run("ssh root\@$guest \"echo BOOTPROTO=\\'dhcp\\' > /etc/sysconfig/network/ifcfg-$nic\"");

```